docs/tsg_pvs.md

```diff
@@ -136,6 +136,79 @@ Deleting OpenEBS Persistent Volume and Persistent Volume Claim did not change th
 To reclaim the space currently, you must perform a manual delete `rm -rf` of the files in */var/openebs* (or whichever path the storage pool is created on). For more information, see [this](https://github.com/openebs/openebs/issues/1436).
 
 
+## Recover data from Jiva replica
+
+This document contains notes on how to recover data from a backedup Jiva replica files. OpenEBS Jiva volumes, save the data in /var/openebs/pvc-id/. All the replica's contain identical data. Before performing a cluster re-build, it suffices to have a backup of data from one of the replica's /var/openebs/pvc-id/ .
+
+These steps are required where replicas could get scheduled onto a node where the data doesn't exist.
```

Comment:
The following procedure helps replicas getting scheduled on nodes where data does not exist.